include/apps/testkeys.h

```diff
@@ -1,3 +1,187 @@
+/*
+ * Copyright 1998-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+static unsigned char dsa512_priv[] = {
```

Comment:
All these constants should really be const and have no business being in a header.

Reply:
Yeah.  Would it be better if I called it `testkeys.inc` ?

Reply:
I moved `include/apps/testkeys.h` to 'apps/testkeys.inc'

Reply:
I'd just add a testkeys.c file with a sane api and provide the prototypes either in speed.c (the only consumers) or apps.h. I can make a PR once this is merged.